Mathlib/Analysis/SpecialFunctions/ContinuousFunctionalCalculus/Rpow/Basic.lean

```diff
@@ -73,6 +74,18 @@ variable {A : Type*} [PartialOrder A] [NonUnitalRing A] [TopologicalSpace A] [St
   [Module ℝ A] [SMulCommClass ℝ A A] [IsScalarTower ℝ A A]
   [NonUnitalContinuousFunctionalCalculus ℝ≥0 A (0 ≤ ·)]
 
+variable {B : Type*} [PartialOrder B] [NonUnitalRing B] [TopologicalSpace B] [StarRing B]
+  [Module ℝ B] [SMulCommClass ℝ B B] [IsScalarTower ℝ B B]
+  [NonUnitalContinuousFunctionalCalculus ℝ≥0 B (0 ≤ ·)]
+  [NonUnitalContinuousFunctionalCalculus ℝ≥0 (A × B) (0 ≤ ·)]
+
+variable {ι : Type*} {C : ι → Type*} [∀ i, PartialOrder (C i)] [∀ i, NonUnitalRing (C i)]
+  [∀ i, TopologicalSpace (C i)] [∀ i, StarRing (C i)]
+  [∀ i, Module ℝ (C i)] [∀ i, SMulCommClass ℝ (C i) (C i)] [∀ i, IsScalarTower ℝ (C i) (C i)]
+  [∀ i, NonUnitalContinuousFunctionalCalculus ℝ≥0 (C i) (0 ≤ ·)]
+  [NonUnitalContinuousFunctionalCalculus ℝ≥0 (∀ i, C i) (0 ≤ ·)]
```

Comment:
Since these `variable` blocks cause big slowdowns with regard to elaboration (which we need to fix separately), can you just put them locally on the declaration (or in a mini section around each declaration)? Or if it makes more sense, group all the `pi` and all the `prod` results in this file in their own sections.